mock-identity-system/src/main/resources/application-local.properties

```diff
@@ -65,3 +65,7 @@ crypto.PrependThumbprint.enable=true
 ## ------------------------------------------- Integrations ------------------------------------------------------------
 #Mock IDA integration props
 mosip.esignet.mock.authenticator.ida.otp-channels=email,phone
+
+##---------------------------------Sunbird-RC Plugin Configurations------------------------------------------------------
+
+mosip.esignet.authenticator.sunbird-rc.auth-factor.kba.field-details={{"id":"policyNumber", "type":"text", "format":""},{"id":"fullName", "type":"text", "format":""},{"id":"dateOfBirth", "type":"date", "format":"dd/mm/yyyy"}}
```

Comment:
can you remove sunbird references. and do we have policyNumber present in mock identity data? if not lets remove this field and use any other existing field.

Reply:
removed